data/items/items-gmg.json

```diff
@@ -824,7 +824,7 @@
 						"unit": "minute"
 					},
 					"components": [
-						"{@action Interact}"
+						"({@action Interact})"
```

Comment:
As a note, it's written/phrased this way here: https://2e.aonprd.com/Equipment.aspx?ID=612